CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Bug Fixes
+
+- Added Particle System and IMGUI modules as a dependency.
```

Comment:
@lopezt-unity can we make particle system an optional dependency?

Reply:
Not really as without it you can't compile.

Reply:
I added this as an optional dependency, it should be lighter.
Thanks Antoine for the help on this one.